### ISSUE TITLE
[doc] multiprocessing.Queue fails intermittently with "Broken pipe"

### ISSUE THREAD

**Issue by cb6f81bf-6992-4dca-b34a-ef5fc052bce1** [python/cpython#82742]
BPO | [38561](https://bugs.python.org/issue38561)
--- | :---
Nosy | @iritkatriel
Files | <li>[test.py](https://bugs.python.org/file48674/test.py "Uploaded as text/plain at 2019-10-23.01:40:48 by Samuel Grayson")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-10-23.01:40:48.912>
labels = ['type-bug', '3.9', '3.10', '3.11', 'library', 'docs']
title = '[doc] multiprocessing.Queue fails intermittently with "Broken pipe"'
updated_at = <Date 2022-01-19.10:16:13.205>
user = 'https://bugs.python.org/SamuelGrayson'
```

bugs.python.org fields:
```python
activity = <Date 2022-01-19.10:16:13.205>
actor = 'iritkatriel'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation', 'Library (Lib)']
creation = <Date 2019-10-23.01:40:48.912>
creator = 'Samuel Grayson'
dependencies = []
files = ['48674']
hgrepos = []
issue_num = 38561
keywords = []
message_count = 2.0
messages = ['355204', '410932']
nosy_count = 3.0
nosy_names = ['docs@python', 'Samuel Grayson', 'iritkatriel']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue38561'
versions = ['Python 3.9', 'Python 3.10', 'Python 3.11']
```

</p></details>

**Comment by slateny** [comment]
Related to #80025 (and possibly the referenced #91185 and #31913)

**Comment by maggyero** [comment]
@JelleZijlstra, could you also close this issue now that it has been fixed in PR #31913?
